images/buildkit/tests/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "version" {
+  digest = var.digest
+  script = "docker run --rm $IMAGE_NAME --version"
+}
```

Comment:
We've been starting to forgo these superficial checks when we have more substantial testing, and I think the below clearly suffices!